changelog.txt

```diff
@@ -1,5 +1,28 @@
+* New: Filter for enabling/disabling enrolment related background jobs [#3182](https://github.com/Automattic/sensei/pull/3182)
+* New: Add "Date Completed" field to the Learner Management page [#3064](https://github.com/Automattic/sensei/pull/3064) 👏 @timwiel
+* New: Add support for translating strings in JavaScript files [#3219](https://github.com/Automattic/sensei/pull/3244)
+* New: Bundle WordPress components CSS [#3226](https://github.com/Automattic/sensei/pull/3226)
+* Tweak: Set up webpack for all JS and CSS builds [#3056](https://github.com/Automattic/sensei/pull/3056), [#3238](https://github.com/Automattic/sensei/pull/3238)
+* Tweak: Add Sensei and WCPC version directly in usage stats [#3094](https://github.com/Automattic/sensei/pull/3094)
+* Tweak: Remove old setup notices in favour of the new wizard [#3155](https://github.com/Automattic/sensei/pull/3155)
+* Tweak: Remove mailing list signup modal in favour of the setup wizard [#3167](https://github.com/Automattic/sensei/pull/3167)
+* Tweak: Include email classes in autoloader [#3098](https://github.com/Automattic/sensei/pull/3098)
+* Tweak: Add precommit hooks to run linters [#3116](https://github.com/Automattic/sensei/pull/3116), [#3196](https://github.com/Automattic/sensei/pull/3196)
```

Comment:
I don't think we want this in the change log, do we?

Reply:
https://github.com/Automattic/sensei/pull/3261/commits/7d1c07a6ff37a838877bd53dc87ed70065d2a96a